ansible/library/generate_golden_config_db.py

```diff
@@ -344,6 +344,24 @@ def update_dns_config(self, config):
         else:
             return config
 
+    def generate_ft2_golden_config_db(self):
+        """
+        Generate golden_config for FT2 to enable FEC.
+        **Only PORT table is updated**.
+        """
+        SUPPORTED_TOPO = ["ft2-64"]
+        if self.topo_name not in SUPPORTED_TOPO:
+            return "{}"
+        SUPPORTED_PORT_SPEED = ["100000", "200000", "400000", "800000"]
```

Comment:
do we need all these speeds

Reply:
Yes, I think so. Actually, for this particular topo, all links are 800Gb.